RealmSwift-swift1.2/Realm.swift

```diff
@@ -376,7 +428,32 @@ public final class Realm {
     public func objectForPrimaryKey<T: Object>(type: T.Type, key: AnyObject) -> T? {
         return unsafeBitCast(RLMGetObject(rlmRealm, type.className(), key), Optional<T>.self)
     }
-
+    
+    /**
+    This method is useful only in specialized circumstances, for example, when building
+    components that integrate with Realm. If you are simply building an app on Realm, it is
+    recommended to use the typed method `objectForPrimaryKey(type:key:)`.
+    
+    Get a dynamic object with the given class name and primary key.
+    
+    Returns `nil` if no object exists with the given class name and primary key.
+    
+    This method requires that `primaryKey()` be overridden on the given subclass.
+    
+    :see: Object.primaryKey()
+    
+    :warning: This method is useful only in specialized circumstances.
+    
+    :param: className  The class name of the object to be returned.
+    :param: key        The primary key of the desired object.
+    
+    :returns: An object of type `DynamicObject` or `nil` if an object with the given primary key does not exist.
+    
+    :nodoc:
+    */
+    public func dynamicObjectForPrimaryKey(className: String, key: AnyObject) -> DynamicObject? {
```

Comment:
Would like to hear everyone's input on this, not opposed to change but this does make difference more clear.